src/main/java/org/jenkinsci/plugins/docker/workflow/WithContainerStep.java

```diff
@@ -134,8 +136,29 @@ private static void destroy(String container, Launcher launcher, Node node, EnvV
             } else {
                 listener.error("Failed to parse docker version. Please note there is a minimum docker version requirement of v1.3.");
             }
-            
-            container = dockerClient.run(env, step.image, step.args, ws, Collections.singletonMap(ws, ws), envReduced, dockerClient.whoAmI(), /* expected to hang until killed */ "cat");
```

Comment:
BTW I plan to add to this default map to fix [JENKINS-27152](https://issues.jenkins-ci.org/browse/JENKINS-27152) so there may be a minor merge conflict.